src/schema/schema_create.c

```diff
@@ -621,6 +621,12 @@ __create_colgroup(WT_SESSION_IMPL *session, const char *name, bool exclusive, co
 
         WT_ERR(__wt_config_collapse(session, cfg, &cgconf));
 
+        /* Add a 5 second wait to simulate schema create slowness. */
+        struct timespec tsp;
+        tsp.tv_sec = 5;
+        tsp.tv_nsec = 0;
+        __wt_timing_stress(session, WT_TIMING_STRESS_CREATE_SLOW, &tsp);
```

Comment:
I don't see a test that uses `WT_TIMING_STRESS_CREATE_SLOW`. How does the testing for this PR work?

I also personally prefer testing through having optional abort() calls as it was done in https://github.com/wiredtiger/wiredtiger/pull/12783 since it eliminates the possibility of not copying the table to a different directory in time + if we anyhow leave it enabled in production, it would be shown immediately which is better in my opinion.